io_ogre/util.py

```diff
@@ -66,6 +67,100 @@ def mesh_tool_parameters():
 def mesh_tool_version():
     return mesh_tool_parameters()[1]
 
+# Calls OgreMeshUpgrader to generate the LOD levels
+def lod_create(infile):
+    #exe = config.get('OGRETOOLS_MESH_UPGRADER')
+    #exe = r"D:\OGRE2\Built\ogre-1.11.6\bin\OgreMeshUpgrader.exe"
+    exe = r"D:\OGRE2\Dev\OgreMeshUpgrader\bin\Release\OgreMeshUpgrader.exe"
```

Comment:
debug remainder?

Reply:
Big oops, thanks for catching that.

Anyway I also realized that I'n not taking OgreMeshTool (Ogre v2) into account, so I will have to do other modifications.